padd.sh

```diff
@@ -643,14 +643,14 @@ PrintNetworkInformation() {
       CleanPrintf "%s\e[0K\\n" "${dhcp_info}"
     fi
   else
-    CleanEcho "${bold_text}NETWORK ========================================================================${reset_text}"
+    CleanEcho "${bold_text}NETWORK =======================================================================${reset_text}"
     CleanPrintf " %-10s%-19s\e[0K\\n" "Hostname:" "${full_hostname}"
     CleanPrintf " %-10s%-19s %-10s%-29s\e[0K\\n" "IPv4 Adr:" "${IPV4_ADDRESS}" "IPv6 Adr:" "${IPV6_ADDRESS}"
     CleanEcho "DNS ============================================================================"
```

Comment:
there is one redundant `=` (equals sign character) that causes a NEWLINE

Reply:
In original padd.sh it got 10 lines where it has extra "=" character that causes NEWLINE. 
This pull is just a small correction for those who are having issues.

Reply:
@christianboyle has a better suggested fix regarding the new CleanEcho() function (line 569) that allows screen to property use 60 characters wide.
<code>
CleanEcho() { echo -e "${ceol}$1" }
</code>

Reply:
Does the order of the "tput el" (via ${ceol}) actually matter?
Because intuitively I would have simply removed the extra space after $1:
`CleanEcho() { echo -e $1"${ceol}" }`